rustworkx/rustworkx.pyi

```diff
@@ -47,6 +47,7 @@ class JSONSerializationError(Exception): ...
 class FailedToConverge(Exception): ...
 class InvalidMapping(Exception): ...
 class GraphNotBipartite(Exception): ...
+class GreedyStrategy: ...
```

Comment:
Since the pyclass isn't configured to allow python subclassing of the enum (which is to correct choice) we need to decorate the stub with `@final`  (https://docs.python.org/3/library/typing.html#typing.final) to indicate this:

```suggestion
@final
class GreedyStrategy: ...
```

Reply:
Oh, I see the other stub test failure is that the class attributes for the variants aren't included either. I think the syntax might be something like:

```python
@final
class GreedyStrategy: ...
    Degree: ...
    Saturation: ...
```

But that might not work, I'd have to refer to python typing documentation. @IvanIsCoding do you happen to know off the top of your head?

Reply:
@mtreinish that is correct, you need to include each method and attribute of a class. You can look at PyDiGraph with multigraph, attrs etc. If there is no specified type you can use `variable: Any`